### DISCUSSION TITLE
Intégrer l'Aide au Retour à l'Emploi (ARE)

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2625] [Idées]
https://www.shine.fr/blog/chomage-auto-entrepreneur/#comment-est-calculee-lare-avec-une-activite-dauto-entrepreneur

**Comment by mquandalle** [comment]
@LucasDetre est-ce un sujet que vous traitez dans [Estime](https://beta.gouv.fr/startups/estime.html) ?

**Comment by johangirod** [comment]
On pourrait utiliser l'API connecté de pôle emploi qui permet d'obtenir directement les indemnisation (dont l'ARE) et afficher l'impact qu'aurait une création d'entreprise sur ces dernières.

https://pole-emploi.io/data/api/pole-emploi-connect/indemnisations

**Comment by Juliedelevoie** [comment]
Pourriez-vous ajouter une partie sur "êtes-vous demandeur d'emploi indemnisé ?" avec calcul si maintien de l'ARE pour les demandeurs d'emploi, ou "maintien RSA" pour les BRSA ?
Avec une simulation de prévisionnel de trésorerie simple.
https://bpifrance-creation.fr/encyclopedie/aides-a-creation-a-reprise-dentreprise/aides-sociales-financieres/aides-pole-emploi
